external/qunit-assert-classes/qunit-assert-classes.js

```diff
@@ -0,0 +1,37 @@
+( function( QUnit ) {
+	function check( element, classes, stateVal, message ) {
+		var result,
+			classArray = classes.split( " " ),
+			missing = [],
+			found = [];
+
+		if ( element.jquery && element.length > 1 ) {
+			$.error( "Class checks can only be performed on a single element on a collection" );
+		}
+		element = element.jquery ? element[ 0 ] : element;
+		message = message || "Element must " + ( stateVal ? "" : "not " ) + "have classes";
+		if ( element.getAttribute( "class") ) {
+			classArray.forEach( function( value ) {
```

Comment:
You should set the `es3` option when running jshint to catch that. Not sure if it actually does, worth a try.